src/engine/openspaceengine_lua.inl

```diff
@@ -288,6 +292,71 @@ int downloadFile(lua_State* L) {
     return 0;
 }
 
+/**
+* \ingroup LuaScripts
+* createPixelImage():
+* Creates a one pixel image with a given color and returns the p
+*/
+int createPixelImage(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 2, "lua::createPixelImage");
+
+    const std::string& name = ghoul::lua::value<std::string>(L, 1);
+    const ghoul::Dictionary& d = ghoul::lua::value<ghoul::Dictionary>(L, 2);
+
+    using namespace openspace::documentation;
```

Comment:
Maybe you can put a @TODO comment in here that we want to clean this up with a more direct use of the Verifier in the future